### PR TITLE
Temp fix to the observer to survive network loss

### PR DESCRIPTION
**Pull request by wcgcyx**
This PR introduces a temp fix to the observer so it can recover from network loss. It will retry once the network error appears. 
Some updates to the terraform script and also the logging package so it displays caller (for easy debug).

### CODE REVIEW

File: services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go [hunk 1]
```diff
@@ -145,25 +146,33 @@ func (o *ObserverImplV1) StopObserve() error {
...
 
-	observer, err := NewSFCBridgeObserver(chainID.String(), addr.String(), sfc, o.mq, end)
-	if err != nil {
-		logging.Error(err.Error())
-		return
+		observer, err := NewSFCBridgeObserver(chainID.String(), addr.String(), sfc, o.mq, end)
+		if err != nil {
+			logging.Error(err.Error())
+			return
+		}
```
Comment (ermyas) [inline services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go:167]: Is there a reason we would repeat these steps (lines 150-167) if the observer fails to start on line 168? Could the for loop start after line 167 instead?

File: services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go [hunk 2]
```diff
@@ -145,25 +146,33 @@ func (o *ObserverImplV1) StopObserve() error {
...
+		if observer.Start() == nil {
+			break
+		} else {
+			logging.Warn("Error in observing event. Retry in 3 seconds...")
+			chain.Close()
+			time.Sleep(3 * time.Second)
```
Comment (ermyas) [inline services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go:173]: should we have a finite number of retries?

**Review by ermyas — APPROVED** [review]
LGTM, minor comments added

File: services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go (hunk 1 above)
Comment (wcgcyx) [reply services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go:167]: The idea here is that if a network loss appears then we need to basically dial again by closing the old connection, open a new one + refresh the sfc and observer with the new ethclient instance. If we just loop after line 167, my suspicion is that the observer will just reuse the old connection.

File: services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go (hunk 2 above)
Comment (wcgcyx) [reply services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go:173]: At the moment, it fails if the retry failed. So say we have a network loss, we will wait 3 seconds and try to dial again and if the retry failed, the program will exit at line 153.

File: services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go (hunk 1 above)
Comment (ermyas) [reply services/relayer/internal/msgobserver/eth/observer/observer_impl_v1.go:167]: Ah, I assumed the `observer.Start()` call here would execute on a separate goroutine, but instead, it is a blocking and long-running operation so resetting the connection makes sense to me now.

**Pull request merged**